2018-12-03 16:29:47 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Column "COMPUTADOR" not found; SQL statement:
select computador [42122-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:853)
	at org.h2.command.Parser.prepareCommand(Parser.java:259)
	at org.h2.engine.Session.prepareLocal(Session.java:561)
	at org.h2.engine.Session.prepareCommand(Session.java:502)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1203)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
2018-12-03 17:23:56 database: wrong user or password; user: "SA1"
org.h2.message.DbException: Wrong user name or password [28000-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:104)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:115)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.test(WebApp.java:885)
	at org.h2.server.web.WebApp.process(WebApp.java:220)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: Wrong user name or password [28000-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 19 more
2018-12-03 17:24:05 jdbc[9]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:184)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:126)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:99)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.test(WebApp.java:885)
	at org.h2.server.web.WebApp.process(WebApp.java:220)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:169)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-193]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 14 more
2019-09-12 14:52:17 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Value too long for column "HA BINARY(255)": "X'aced0005737200186d6f64656c732e486973746f7269636f446541636573736f4d4d1eca74ac9d6302000c49000c686f7261446541636573736f49000e6d69... (421)"; SQL statement:
insert into Usuario (id, HA, email, matricula, nome, senha, sobrenome, tipo) values (null, ?, ?, ?, ?, ?, ?, ?) [22001-193]
2019-09-12 15:01:35 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Value too long for column "HA BINARY(255)": "X'aced0005737200186d6f64656c732e486973746f7269636f446541636573736f4d4d1eca74ac9d6302000c49000c686f7261446541636573736f49000e6d69... (421)"; SQL statement:
insert into Usuario (id, HA, email, matricula, nome, senha, sobrenome, tipo) values (null, ?, ?, ?, ?, ?, ?, ?) [22001-193]
2019-09-12 15:04:18 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Value too long for column "HA BINARY(255)": "X'aced0005737200186d6f64656c732e486973746f7269636f446541636573736f4d4d1eca74ac9d6302000c49000c686f7261446541636573736f49000e6d69... (421)"; SQL statement:
insert into Usuario (id, HA, email, matricula, nome, senha, sobrenome, tipo) values (null, ?, ?, ?, ?, ?, ?, ?) [22001-193]
